graphql_api/helpers/connection.py

```diff
@@ -205,6 +206,14 @@ class DictCursorPaginator(CursorPaginator):
     if the dict access fails then it throws an exception, although it would be a different
     """
 
+    def decode_cursor(self, cursor):
+        try:
+            orderings = b64decode(cursor.encode("ascii")).decode("utf8")
+            orderings = orderings.split(self.delimiter)
+            return [None if ordering == "\x1f" else ordering for ordering in orderings]
```

Comment:
Damn, weird bug/good find

Reply:
Nit, make a var for \x1f since we use in multiple places and to explain what the val is/why it's needed